modules/visual_script/register_types.cpp

```diff
@@ -116,7 +113,6 @@ void register_visual_script_types() {
 	ClassDB::set_current_api(ClassDB::API_EDITOR);
 	GDREGISTER_CLASS(vs_bind::VisualScriptEditor);
 	ClassDB::set_current_api(ClassDB::API_CORE);
-	vs_editor_singleton = memnew(vs_bind::VisualScriptEditor);
 	Engine::get_singleton()->add_singleton(Engine::Singleton("VisualScriptEditor", vs_bind::VisualScriptEditor::get_singleton()));
```

Comment:
Note that this is the other singleton of the same class/type that gets actually used